test/fixtures/dialects/ansi/double_quote_escapes.yml

```diff
@@ -12,25 +12,25 @@ file:
       - select_clause_element:
           numeric_literal: '1'
           alias_expression:
-            keyword: as
+            alias_operator: as
```

Comment:
I think the desired output here would be
```yaml
alias_operator:
  keyword: as
```
otherwise capitalisation rules may not work properly for it